status-im/status-react
# Add ST to assets list

fixes #6624 

### Summary:

Adds ST token to list of available assets.

### Steps to test:

- Open Status
- Go to Manage Assets, add ST.
- You'll see that you can now add ST.

status: ready
